content/applications/general/iot/config/troubleshooting.rst

```diff
@@ -343,6 +343,66 @@ Save`.
              :align: center
              :alt: Receipt format using naming convention: EpsonTMU220B__IMC_LDV_LDH_SCALE35__.
 
+DYMO LabelWriter 450 DUO print delay
+------------------------------------
+
+The DYMO LabelWriter 450 DUO has a known delay in printing with the :abbr:`IoT (Internet of Things)`
+box while using the label function. This is due to the printer being set up as a *Local RAW
```

Comment:
technically false. If you check back the video it does not print at all if the raw driver is used (which is the default one set when cups detect the printer